src/PerfectGym.AutomergeBot/Notifications/UserNotifications/UserNotifier.cs

```diff
@@ -110,15 +122,93 @@ public void NotifyAboutOpenPullRequests(IEnumerable<PullRequest> filteredPullReq
             }
         }
 
+        public void SendDirectMessageAboutOpenPullRequests(IEnumerable<PullRequest> pullRequests)
+        {
+            foreach (var notify in GetNotificationsAboutOpenPullRequest(pullRequests))
+            {
+                using (var client = _slackClientProvider.Create())
+                {
+                    SendDirectMessageAboutPullRequests("Your open pull requests:", notify.Login, notify.OwnPullRequests, client);
+                    Thread.Sleep(TimeSpan.FromSeconds(1));
+
+                    SendDirectMessageAboutPullRequests("The pull requests waiting for your review:", notify.Login, notify.PullRequestsToReview, client);
+
+                    Thread.Sleep(TimeSpan.FromSeconds(1));
+
+                }
+
+            }
+        }
+
+        private void SendDirectMessageAboutPullRequests(string headOfmessage, string login, IEnumerable<PullRequest> pullRequests, ISlackClient client)
+        {
+            if (pullRequests!=null && pullRequests.Any())
+            {
+                var message = _messageProvider.CreatePullRequestMessage(
+                    headOfmessage,
+                    login,
+                    pullRequests
+                );
+
+                try
+                {
+                    client.SendMessageToUser(message, login);
+                }
+                catch (SlackApiErrorException e)
+                {
+                    _logger.LogError(e, "Failed notifying user {User}", login);
+                }
+            }
+        }
+
+        private IEnumerable<NotificationsAboutOpenPullRequest> GetNotificationsAboutOpenPullRequest(IEnumerable<PullRequest> pullRequests)

```

Comment:
Nazwa metody powinna być zgodna z tym co faktycznie metoda robi.
```suggestion
        private IEnumerable<NotificationsAboutOpenPullRequest> CreateNotificationsAboutOpenPullRequest(IEnumerable<PullRequest> pullRequests)
```

Reply:
Użycie przedrostka Create może sugerować, że notyfikacja się wykona.
Metoda Get opisuje działanie metody, czyli zwraca notyfikacje, to że tam się odbywa tworzenie instancji obiektu jest drugorzędnym szczegółem implementacyjnym.